Restructure short header section (#4407)
quicwg/base-drafts: draft-ietf-quic-transport.md

~~~~~
{: #fig-short-header title="Short Header Packet Format"}

> This version of QUIC uses a short packet header for only 1-RTT
> packets. Additionally, 1-RTT packets in this version of QUIC can only use a
> short packet header

Reviewer: ```suggestion
This version of QUIC only uses short packet headers for 1-RTT
packets. 1-RTT keys in this version of QUIC are only used with
short packet headers.
```
